### PR TITLE
Tesb416 217

### PR DESCRIPTION
**Pull request by chrisguter**

### CODE REVIEW

File: web app/src/main/java/de/tinf15b4/kino/web/ui/SmartCinemaUi.java [hunk 1]
```diff
@@ -125,7 +128,8 @@ private Component createHeader() {
         search.addComponent(searchField);
         search.addComponent(searchButton);
         searchButton.addClickListener(
-                e -> getNavigator().navigateTo(SearchResultsView.VIEW_NAME + "/" + searchField.getValue()));
+                e -> getNavigator().navigateTo(
+                        SearchResultsView.VIEW_NAME + "/" + searchField.getValue()));
```
Comment (flitzpiepe96) [inline web app/src/main/java/de/tinf15b4/kino/web/ui/SmartCinemaUi.java]: Seems like you are using some wring formatting. There was no change in this line as far as I can see, but still there is a new formatting

File: web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java [hunk 2]
```diff
@@ -58,75 +57,200 @@ public void enter(ViewChangeEvent event) {
                 HorizontalLayout content = new HorizontalLayout();
 
                 // Image
-                Component image = new Image(null, new ExternalResource(userBean.getRestClient().getMoviePictureUrl(m)));
-                image.setHeight("400px");
-                image.setId("movieImage_" + m.getId());
+                Component image = new Image(null,
+                        new ExternalResource(userBean.getRestClient().getMoviePictureUrl(m)));
+                image.setWidth("250px");
+                image.setId("movieDetailImage");
```
Comment (flitzpiepe96) [inline web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java]: More formatting. This seems to have happened across all files you touched

File: web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java [hunk 3]
```diff
@@ -58,75 +57,200 @@ public void enter(ViewChangeEvent event) {
...
+            ratingsForm.addComponent(createRatingEntry(rm));
+
+        }
+
+        ratingsForm.setId("ratingsForm");
+
+        return ratingsForm;
+    }
+
+    private VerticalLayout createRatingEntry(RatedMovie rm) {
```
Comment (flitzpiepe96) [inline web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java]: This seems (and actually should) to be quite  similar toe the ratings in cinema view. I think it should be possible to extract this duplicate code to some utility class. I think we did something like that for the favorite buttons

Comment (rgcjonas) [inline web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java]: Suggestion: Wait a little while and see if the metrics tool catches it. Then we could use this as a refactoring for this week

Comment (flitzpiepe96) [inline web app/src/main/java/de/tinf15b4/kino/web/views/MovieView.java]: +1

**Pull request merged**